src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -423,6 +423,61 @@
             </div>
           </div>
 
+        <!-- Quotas -->
+        <div>
+          <legend i18n>Quotas</legend>
+
+          <!-- Max Bytes -->
+          <div class="form-group"
+               [ngClass]="{'has-error': form.showError('max_bytes', formDir)}">
+            <label class="control-label col-sm-3"
+                   for="max_bytes">
+              <ng-container i18n>Max bytes</ng-container>
+            </label>
+            <cd-helper>
+              <span i18n>Leave it blank or specify 0 to disable this quota.</span>
+              <br>
+              <span i18n>A valid quota should be greater then 0.</span>
+            </cd-helper>
+            <div class="col-sm-9">
+              <input class="form-control"
+                     id="max_bytes"
+                     name="max_bytes"
+                     type="text"
+                     formControlName="max_bytes"
+                     i18n-placeholder
+                     placeholder="e.g., 10GiB"
+                     defaultUnit="GiB"
+                     cdDimlessBinary>
+            </div>
+          </div>
+
+          <!-- Max Objects -->
+          <div class="form-group"
+               [ngClass]="{'has-error': form.showError('max_objects', formDir)}">
+            <label class="control-label col-sm-3"
+                   for="max_objects">
+              <ng-container i18n>Max objects</ng-container>
+            </label>
+            <cd-helper>
```

Comment:
cd-helper should be inside the label, after the ng-container.

Reply:
Thanks! done